song-server/src/main/java/org/icgc/dcc/song/server/repository/UploadRepository.java

```diff
@@ -24,8 +24,14 @@
 import org.skife.jdbi.v2.sqlobject.SqlQuery;
 import org.skife.jdbi.v2.sqlobject.SqlUpdate;
 import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper;
+import org.skife.jdbi.v2.sqlobject.stringtemplate.UseStringTemplate3StatementLocator;
+import org.skife.jdbi.v2.unstable.BindIn;
```

Comment:
I notice that the BindIn class is in the "unstable" package. Have we checked to see how "unstable" it really is?